doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,283 @@
+variable information as outlined below. If you do not find your variable
+here, you have to provide the variable information in an additional file
+that will have to be placed in he folder ``/esmvalcore/cmor/tables/custom/``.
+
+The file that you have to create for a custom variable needs to follow these
+guidelines:
+
+- Provide information about the SOURCE (for all observations and reanalysis it
+  was decided by the ESMValTool development team that this is set to ``CMIP5``;
+  it basically describes the framework in which the individual variables are
+  associated with realms; it might not be necessary to provide this information
+  here, but before we know for sure that we can leave it out, please just
+  provide it);
+- Provide the ``variable_entry``;
+- Provide the ``modeling_realm``;
+- Provide the variable attributes, but **do not** provide a standard name! The
+  custom variables will only be read by Iris if this information is not
+  provided! Necessary variable attributes are: ``units``, ``cell_methods``,
+  ``cell_measures``, ``long_name``, ``comment``.  
+- Provide some additional variable attributes. Necessary additional variable
+  attributes are: ``dimensions``, ``out_name``, ``type``. There are also
+  additional variable attributes that can be defined here (see the already
+  available cmorizers). 
+
+Use the file structure as given in the files that already exist, and save the
+file with the extension ``<short_name>.dat``.
+
+2. Configuration file
+=====================
+
+It is important to have your configuration file defined correctly, to be sure
+that your data files are found by the ESMValTool, and that the cmorized data
+files are written in the correct folders. See Section **link to be added** for
+an example of a configuration file, that has to be filled out with the paths
+information of your specific machine.
+
+3. Store your dataset in the right place
+========================================
+
+For the ESMValTool to find the data set that it needs, a specific folder
+structure is defined. You have your path that points towards the
+observations. That path is defined in your configuration file (see Section 2)
+under ``RAWOBS``. From there you need the subdirectories ``Tier1``, ``Tier2``
+and ``Tier3``. In the folder of your choice, you need a folder with the name of
+the dataset, and in that folder you put the data files with the raw
+observations (as you can download them from the observations data source
+webpage).
+
+4. Reformat your data set to meet CMOR standard - script
+========================================================
+
+To make sure that your dataset is in the correct format for Iris to work with
+it, you most likely will have to process it with a cmorizing script. In the
+rare, but possible, case that your dataset is in the correct format already, it
+is advisable to run it through a cmorizing script anyway, to be sure the file
+name is correctly defined for the ESMValTool to find the file. 
+
```

Comment:
The rest of this section is far too detailed in my opinion. 
I would simply write that a cmorizer script can be created following the existing examples, either Python or NCL.
Mention that utility functions for the cmorization are available in both languages (`utilities.ncl`, `utilities.py`). 
Please also add a Sphinx note to warn the user that Python should be preferred since NCL support will terminate soon.
```
.. note::
  Text here
```

Reply:
Ok, I shortened it a bit, and tried to add the Sphinx note. It does not show up properly yet, don't know why.